### PR TITLE
Create CICD in Github Action To apply Octavia configs

### PR DESCRIPTION
**Pull request by leanbc2**
Create CICD in Github Action To apply Octavia configs

### CODE REVIEW

File: octavia-configs/sources/revenue_db/state_33350466-92ca-45ae-a455-346578b37b9d.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+configuration_hash: 4a10876d297c898ba6375312041427bef2f2b6e7e1a1d75e79c3cb4e93ab3b46
```
Comment (SardorQogita) [inline octavia-configs/sources/revenue_db/state_33350466-92ca-45ae-a455-346578b37b9d.yaml]: why do we need state files?

File: octavia-configs/destinations/snowflake/configuration.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+definition_type: destination
+definition_id: 424892c4-daac-4491-b35d-c6688ba547ba
+definition_image: airbyte/destination-snowflake
+definition_version: 0.4.41
+configuration:
+  host: il13848.eu-central-1.snowflakecomputing.com
+  role: ACCOUNTADMIN
+  schema: REVENUE_DB_PUBLIC
+  database: RAW_TEST
+  username: leandrocantos
```
Comment (SardorQogita) [inline octavia-configs/destinations/snowflake/configuration.yaml]: why not create an airbyte user instead?

File: octavia-configs/destinations/snowflake/configuration.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+resource_name: Snowflake
```
Comment (SardorQogita) [inline octavia-configs/destinations/snowflake/configuration.yaml]: Shouldn't we name it Snowflake_test_revenue_db_public?

File: octavia-configs/sources/revenue_db/state_33350466-92ca-45ae-a455-346578b37b9d.yaml (hunk 1 above)
Comment (leanbc2) [reply octavia-configs/sources/revenue_db/state_33350466-92ca-45ae-a455-346578b37b9d.yaml]: They are actually not needed. I will remove them

**Review by SardorQogita — APPROVED** [review]

**Pull request merged**